File: core/app/datasource/src/main/java/io/openk9/datasource/events/DatasourceEvent.java
package io.openk9.datasource.events;

public record DatasourceEvent(
		String ingestionId,
		long datasourceId,
		String contentId,
		long parsingDate,
		String rawContent,
		String tenantId,
		String[] documentTypes,
		String indexName,
		String errorMessage) {
}

File: js-packages/admin-ui/src/openapi-generated/models/DatasourceEvent.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type DatasourceEvent = {
    ingestionId?: string;
    datasourceId?: number;
    contentId?: string;
    parsingDate?: number;
    rawContent?: string;
    tenantId?: string;
    documentTypes?: Array<string>;
    indexName?: string;
    errorMessage?: string;
};
